app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -320,7 +320,7 @@ def create_vm(args = { })
 
       args = parse_networks(args)
       args = args.with_indifferent_access
-      if args[:image_id].present?
+      if args[:provision_method] == 'image'
```

Comment:
You are right. I only tested this by clicking "Clone" on a Host, obviously that was not enough.